### PR TITLE
Add a nav link to the Gatsby Starters

### PR DESCRIPTION
**Pull request by Jared-Dev**
## Description

I consistently have to do a search via a search engine to get to the
starters. Using the Algolia search ends up taking me to the docs
which will finally get me to the starters. I find the starters to be a
huge benefit to Gatsby and think having a nav link directly to them
would be a benefit to people starting out with Gatsby (or that just
want an easier way to get to them)!

### CODE REVIEW

File: www/src/utils/presets.js [hunk 1]
```diff
@@ -9,8 +9,8 @@ module.exports = {
     sm: `@media (min-width: 550px)`,
     Md: `(min-width: 750px)`,
     md: `@media (min-width: 750px)`,
-    Lg: `(min-width: 1000px)`,
-    lg: `@media (min-width: 1000px)`,
+    Lg: `(min-width: 1050px)`,
```
Comment (sidharthachatterjee) [inline www/src/utils/presets.js]:
```suggestion
    Lg: `(min-width: 1000px)`,
```

Can we avoid changing the breakpoint?

File: www/src/utils/presets.js [hunk 2]
```diff
@@ -9,8 +9,8 @@ module.exports = {
...
+    lg: `@media (min-width: 1050px)`,
```
Comment (sidharthachatterjee) [inline www/src/utils/presets.js]:
```suggestion
    lg: `@media (min-width: 1000px)`,
```

File: www/src/utils/presets.js (hunk 1 above)
Comment (Jared-Dev) [reply www/src/utils/presets.js]: You can, but it causes a horizontal scroll-bar.

**Comment by fk** [comment]
Good call @Jared-Dev!

FWIW, we discussed grouping "Plugins" and "Starters" in a new (already existing) landing page "Ecosystem" (https://www.gatsbyjs.org/ecosystem/) last year, which could replace the current "Plugins" item in the main navigation, and would resolve the space issue — but I can see how it helps having both "Plugins" and "Starters" in the main navigation. I'll bring that question up with the team again!

File: www/src/utils/presets.js (hunk 1 above)
Comment (fk) [reply www/src/utils/presets.js]: I agree with @sidharthachatterjee — we shouldn't adjust the breakpoint constant, but instead opt-out of using it to define when the mobile navigation is hidden/shown, and instead use a bespoke media query (`@media (min-width: [whatever value we actually need]`).

Comment (fk) [reply www/src/utils/presets.js]: On another note, I worked on #12603 over the weekend, which (among other stuff) changes how the main navigation items look, which gains us a bit of space:

image

@sidharthachatterjee #12603 is ready to go — maybe we merge that, and then check how things look here with the "Starters" item added? Happy to merge `master` here.

Comment (Jared-Dev) [reply www/src/utils/presets.js]: Dropping the uppercase does seem to fix the spacing issue even with the breakpoint reset to `1000px` and leaving starters in place.

**Comment by Jared-Dev** [comment]
Not sure how I missed that! 

I didn't even realize Starters existed until @kkemple introduced them to me (I was looking for a TypeScript based setup). I personally would prefer a direct link in the main nav! If you guys have an A/B framework in place however it could be a really interesting test to see how the two options get used though (Plugins + Starters vs. Ecosystem).

**Comment by KyleAMathews** [comment]
I'd rather not add more items to the top nav — there's quite a few there already.

An alternative proposal https://github.com/gatsbyjs/gatsby/issues/12767

**Comment by LekoArts** [comment]
@KyleAMathews Can we close this PR as #12767 was fixed?

**Comment by KyleAMathews** [comment]
Yup! Thanks @LekoArts for following up on this and for others who worked on fixing this issue.

**Pull request closed without merging**